amt/repositories/tasks.py

```diff
@@ -53,6 +67,24 @@ def save(self, task: Task) -> Task:
             raise RepositoryError from e
         return task
 
+    def save_all(self, tasks: Sequence[Task]) -> Sequence[Task]:
+        """
+        Stores the given tasks in the repository or throws a RepositoryException
+        :param tasks: the tasks to store
+        :return: the updated tasks after storing
+        """
+        # TODO: (Christopher) bulk insert via `add_all` does not work. That's why
+        # we do a loop and add tasks individually.
+        for task in tasks:
```

Comment:
Actually the add_all does work when you give a list as parameter. However, the session.refresh does not accept a list as argument. You could use self.session.add_all(tasks) and session.commit and leave out the session.refresh. It is not required to return the (updated) results and with a bulk_insert you probably never use the return results anyway.